src/vitabel/vitals.py

```diff
@@ -1468,64 +1495,79 @@ def cycle_duration_analysis(self):
-        """Determines start and stop of periods with continuous chest compressions.
+        cc_events_channel.attach_label(cc_periods)
 
-        The procedure is implemented as described in `10.1016/j.resuscitation.2021.12.028 <https://doi.org/10.1016/j.resuscitation.2021.12.028>`_ and `10.1016/j.dib.2022.107973 <https://doi.org/10.1016/j.dib.2022.107973>`_.
+    def find_CC_periods_acc(
+            self, 
+            accelerometer_channel: Channel | str = 'cpr_acceleration',
+        ) -> None:  # part of register application
+        """
+        Automatically detects periods of continuous chest compressions.
 
-        Requires a channel 'cpr_acceleration' in the recording, which is the signal of an accelerometry-based feedback sensor for cardiopulmonary resuscitation.
+        The procedure is implemented as described in `10.1016/j.resuscitation.2021.12.028 <https://doi.org/10.1016/j.resuscitation.2021.12.028>` and `10.1016/j.dib.2022.107973 <https://doi.org/10.1016/j.dib.2022.107973>`_.
+        In essence it uses the root mean square of the accelerometer signal of feedback sensor for cardiopulmonary resuscitation to detect the rise in "power" of the signal linked to the alteration by the accelerations of continous chest compressions. 
+
+        Parameters
+        ----------
+        accelerometer_channel
+            The channel containing the accelerometer signal. Defaults to 'cpr_acceleration'. 
 
         Returns
         -------
         None.
-        Adds two labels 'cc_period_start_acc' and 'cc_period_stop_acc' to the recording.
 
+        Attaches an IntervalLabel withe the name ``cc_periods`` to the accelerometer channel.
+        Every entry in the label describes a single period of chest compressions.
+
+        .. SEEALSO::
+            The method is described in `10.1016/j.resuscitation.2021.12.028 <https://doi.org/10.1016/j.resuscitation.2021.12.028>`_ or in the
+            Thesis 'Towards a data-driven cardiac arrest treatment' by Wolfgang Kern in more detail.
+            See https://unipub.uni-graz.at/obvugrhs/content/titleinfo/10138095 for more information.
         """
-        if "cpr_acceleration" not in self.get_channel_names():
+        if isinstance(accelerometer_channel, str):
+            if accelerometer_channel not in self.get_channel_names():
+                logger.error(
+                    f"The specified channel '{accelerometer_channel}' could not be identified."
+                    "Please specify a channel or a string with the name of the channel."
+                )
+                return
+            else:
+                ACC_channel = self.get_channel(accelerometer_channel)
+        elif not isinstance(accelerometer_channel, Channel):
             logger.error(
-                "No Acceleration data found. Can not identify CC-periods via acceleration."
+                "No valid accelerometer channel specified. Can not identify CC-periods via acceleration."
```

Comment:
also should be turned into a proper exception, `raise ValueError( ... )`, no return afterwards

```suggestion
                "No valid accelerometer channel specified. Can not identify CC-periods via acceleration. "
```

Reply:
implemnted via 7020d2cf3232f3978efaaf94980fb795f503d0b2